contributors.js

```diff
@@ -23,5 +23,6 @@ const contributors = [
   { username: 'SameerKhans13', message: 'Happy Hacktoberfest-24 to all!' },
   { username: 'rishi-harti768', message: 'Happy Hacktoberfest guys! cheers'},
   { username: 'Hathaipach', message: 'Happy Hacktoberfest 2024'},
-  { username: 'THE-ASHUTOSH', message: 'Happy Hacktoberfest 2024 all'}
+  { username: 'THE-ASHUTOSH', message: 'Happy Hacktoberfest 2024 all'},
+  { username: 'exilonium', message: 'The element of lie'}
```

Comment:
Hello @exilonium , thanks for this PR. Why this message ? 😅

Reply:
From alt account 

I just wanted something different, I guess it's okay 😁

Reply:
Yes ok.